src/Orleans.Core/Lifecycle/LifecycleSubject.cs

```diff
@@ -47,7 +47,7 @@ public LifecycleSubject(ILogger<LifecycleSubject> logger)
                     Stopwatch stopWatch = Stopwatch.StartNew();
                     await Task.WhenAll(observerGroup.Select(orderedObserver => WrapExecution(ct, orderedObserver.Observer.OnStart)));
                     stopWatch.Stop();
-                    this.logger?.Info(ErrorCode.SiloStartPerfMeasure, $"Starting lifecycle stage {this.highStage} took {stopWatch.ElapsedMilliseconds} Milliseconds");
+                    //this.logger?.Info(ErrorCode.SiloStartPerfMeasure, $"Starting lifecycle stage {this.highStage} took {stopWatch.ElapsedMilliseconds} Milliseconds");
```

Comment:
Why commenting it? I think we should put it as verbose maybe, but definitely not remove it, a tleast not in this PR.

Reply:
I commented this out temporarily, and didn't uncomment it since this is a WIP pr. but this won't be commited if we merge this pr. And there's another pr out there to make the logging less verbose. My plan was to rebase on top of that if that pr got merged. 

sorry for the confusion